test/client/api.js

```diff
@@ -560,6 +560,20 @@ module.exports = config => {
       )
   }
 
+  ClientApi.prototype.sessionDuplicate = function (sessionTokenHex, options) {
+    return tokens.SessionToken.fromHex(sessionTokenHex)
+      .then(
+        function (token) {
+          return this.doRequest(
+            'POST',
+            this.baseURL + '/session/duplicate',
```

Comment:
Purely stylistic nit-picking here, but I find template strings to be generally more readable than concats fwiw. Granted in this case the string is so simple that it makes no difference, but when the interpolation is more complex it really helps. If we wanted to decide on one or the other for everywhere going forward, that''s where my vote would be...